PosNeg.c

```diff
@@ -0,0 +1,20 @@
+//Clarke Littlejohn
+// This is  the Code to see if a number is positive or negative
```

Comment:
You didn't handle floats like the directions said you should. Aside from that, you did a good job. Be careful with directions next time.